### PR TITLE
Allow <C-p> and <C-n> to be used as prev/next when entering a command or search

### LINKED ISSUE #3942: Ctrl-P and Ctrl-N should be equivalent to Up / Down when entering a command or search
See title. I don't think this interferes too much with the VSCode's default Ctrl+P, and if it does for someone, they can disable it (via `handleKeys` or by changing the `when` clause of `extension.vim_ctrl+p`).

### PR DESCRIPTION
**Pull request by J-Fields**
**What this PR does / why we need it**:
Contributes keybindings and corresponding support in code for `<C-p>` and `<C-n>` to act as prev/next when entering a command or search term.

**Which issue(s) this PR fixes**
Fixes #3942

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -223,6 +233,16 @@
         "command": "extension.vim_ctrl+p",
         "when": "suggestWidgetVisible && vim.active && vim.use<C-p>"
       },
+      {
+        "key": "ctrl+p",
+        "command": "extension.vim_ctrl+p",
+        "when": "vim.mode == 'SearchInProgressMode' && vim.active && vim.use<C-p>"
```
Comment (xconverge) [inline package.json]: I think it was a mistake to not have !inDebugRepl in the previous one

Comment (J-Fields) [reply package.json]: Should I add it to the existing ctrl+p binding?

Comment (J-Fields) [reply package.json]: I'm assuming the answer to that is "yes". Resolved.

Comment (xconverge) [reply package.json]: thanks! Sorry to make you assume :)

**Pull request merged**